ow_lander/srv/Grind.srv

```diff
@@ -3,6 +3,7 @@ float32 x               # X coordinate of grinding starting point
 float32 y               # Y coordinate of grinding starting point
 float32 depth           # Desired depth
 float32 length          # Desired length
+bool parallel           # If True, resulting trench is parallel to arm. If False, perpendicular to arm
 float32 ground_position # Z coordinate of ground level in base_link frame
 bool delete_prev_traj   # Set to true to cleanup ~/.ros
```

Comment:
This strikes me as a dangerous parameter, one that should be replaced with a safer way to manage ~/.ros and trajectories.   Recall the bug we had in which a service call that passed true here inadvertently deleted another service's trajectory that was still needed.   Please add a new Jira to address this, unless it already exists.

Reply:
I think you are referring to the **delete_prev_traj**. This actually what happened to me, I thought it would delete only the previous trajectory instead it seems to delete all previous trajectories. Hence, I have added the ticket [API Improvements: Remove delete_prev_traj parameter from all of the services](https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-483)

Reply:
Here it is: https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-486